### PR TITLE
fixes dependencies for gt-svg styling

### PR DESCRIPTION
**Pull request by fgdrf**
With this fix its possible to user svg-graphics in styling again (as described in GeoTools SVG UserGuide : https://docs.geotools.org/latest/userguide/library/render/svg.html). All required batik-dependencies are comming from Eclipse Orbit Repository as OSGi artifacts so these are excluded in pom-libs.xml as documented.

Final result in uDig:

grafik

* see Eclipse CQ's:
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22608 (batik.anim)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22615 (batik.shared.resources)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22611 (batik.awt.util)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22621 (batik.bridge)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22739 (batik.codec)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22304 (batic.constants)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22303 (batik.css)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22617 (batik.dom)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22619 (batik.dom.svg)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22613 (batik.ext)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22618 (batik.gvt)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22301 (batik.i18n)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22616 (batik.parser)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22620 (batik.script)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22615 (batik.shared.resources)
 * (batik.svggen)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22610 (batik.transcoder)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22302 (batik.util)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22614 (batik.xml)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=9587 (org.apache.xalan)
 * https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22305 (org.apache.xmlgraphics)
* removes unused xerces-dependency

Change-Id: Ie8e1fe9776f1fa0fd8b0e6172ba0cb90e8dd6811
Signed-off-by: Frank Gasdorf <fgdrf@users.sourceforge.net>

### CODE REVIEW

**Comment by fgdrf** [comment]
@kassid, @olsmn @sschulz92  you are welcome to review and test ;)

**Comment by fgdrf** [comment]
Tested with Java 1.8 and java 11 ✔️

File: plugins/org.locationtech.udig.libs/pom-libs.xml [hunk 1]
```diff
@@ -604,95 +569,36 @@
...
+					|     |  \- xalan:xalan:jar:2.7.2:compile
+					|     |     \- xalan:serializer:jar:2.7.2:compile
+					|     +- org.apache.xmlgraphics:batik-gvt:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-svggen:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-util:jar:1.10:compile
+					|     |  +- org.apache.xmlgraphics:batik-constants:jar:1.10:compile
+					|     |  \- org.apache.xmlgraphics:batik-i18n:jar:1.10:compile
+					|     +- org.apache.xmlgraphics:batik-xml:jar:1.10:compile
+					|     \- xml-apis:xml-apis-ext:jar:1.3.04:compile
+				-->
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.libs/pom-libs.xml:596]: What is this comment useful for?

Comment (fgdrf) [reply plugins/org.locationtech.udig.libs/pom-libs.xml:596]: it helps to understand which transitive dependencies are not bundled again and might be resolved via "Import-Package" or "Required-Bundle" in MANIFEST.MF from the OSGi platform. just kind of documentation ;)

Comment (sschulz92) [reply plugins/org.locationtech.udig.libs/pom-libs.xml:596]: If you think this is worth to put into the pom-libs.xml I am fine with this PR 👍

**Pull request merged**